langchain/llms/openai.ts

```diff
@@ -87,6 +89,8 @@ type TokenUsage = {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type Kwargs = Record<string, any>;
 
+export type OpenAIRequestConfig = AxiosRequestConfig;
```

Comment:
Wondering if we should nest the axios request config in an object type here to future-proof the third arg of generate for any other things we might want to add in the future, e.g.

```suggestion
export type OpenAIExtraArgs = {
    requestConfig: AxiosRequestConfig
};
```

Reply:
ahhh totally agree. We should also make it optional as well